docs/source/getting_started/installation/index.rst

```diff
@@ -13,11 +13,11 @@ The output should look similar to this:
    % python --version
    Python 3.6.5
 
-Installation for users
-----------------------
-
+Using virtual environments
+--------------------------
+   
 Starfish names its dependencies and lists explicit versions, due to sensitivity to subtle algorithm
```

Comment:
> lists explicit versions

I don't think this is still true today.  There is an option to install the versions we use for CI, but it's no longer mandatory.

Reply:
ah, that's a good point! do we maybe want to just convert this into a strict/nonstrict installation guide? cc @ambrosejcarr @neuromusic

Reply:
Yes, I would add both options to this.